From ea9f7e4cc6fd8c08d175ed7774ed2c5bd11c8ef0 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Mon, 17 Feb 2014 19:37:09 +0000
Subject: Add include of <sys/select.h> for previous patch

Unfortunately while the standard says that <sys/types.h> is the
correct header to get suseconds_t, at least with glibc, that requires
-DXOPEN_SOURCE.  Which is problematic for a public header, because
then all *users* of startup-notification will be required to define
that.

Poking around a bit, it looks like at least with glibc, <sys/select.h>
will give us an unconditional define.

Signed-off-by: Julien Danjou <julien@danjou.info>
---
diff --git a/libsn/sn-monitor.h b/libsn/sn-monitor.h
index 2f639df..cea4e12 100644
--- a/libsn/sn-monitor.h
+++ b/libsn/sn-monitor.h
@@ -28,6 +28,7 @@
 #define __SN_MONITOR_H__
 
 #include <libsn/sn-common.h>
+#include <sys/select.h>
 
 SN_BEGIN_DECLS
 
--
cgit v0.9.0.2-2-gbebe
